zerver/lib/test_runner.py

```diff
@@ -109,8 +109,8 @@ def run_test(test):
     test.setUp()
     try:
         test_method()
-    except unittest.SkipTest:
-        pass
+    except unittest.SkipTest as e:
+        print('Skipped:', e)
```

Comment:
Hmm why do you have two print statements for the test having been skipped, here and below?